content/applications/finance/fiscal_localizations/brazil.rst

```diff
@@ -42,36 +70,46 @@ given to your company.
 #. Select the :guilabel:`Company` option at the top of the page. Then, configure the following
    fields:
 
-   - :guilabel:`Name`.
+   - :guilabel:`Name`
    - :guilabel:`Address` (add :guilabel:`City`, :guilabel:`State`, :guilabel:`Zip Code`,
-     :guilabel:`Country`).
+     :guilabel:`Country`)
 
      - In the :guilabel:`Street` field, enter the street name, number, and any additional address
        information.
      - In the :guilabel:`Street 2` field, enter the neighborhood.
 
-   - :guilabel:`Identification Number` (:guilabel:`CNPJ`, :guilabel:`CPF`).
-   - :guilabel:`Tax ID` (associated with the identification type).
-   - :guilabel:`IE` (State registration).
-   - :guilabel:`IM` (Municipal registration).
-   - :guilabel:`SUFRAMA code` (Superintendence of the Manaus Free Trade Zone - add if applicable).
-   - :guilabel:`Phone`.
-   - :guilabel:`Email`.
+   - :guilabel:`Identification Number` (:guilabel:`CNPJ`, :guilabel:`CPF`)
```

Comment:
I would put the description or options after a semicolon instead of between brackets (it does't bother me that much here, but it does in other similar lists elsewhere in this doc so for the sake of consistency, I'd have them all the same)